src/Service/Font/FontLoader.re

```diff
@@ -55,27 +55,24 @@ let loadAndValidateEditorFont =
           width;
       let {lineHeight, descent, avgCharWidth, maxCharWidth, _}: Revery.Font.FontMetrics.t =
         Revery.Font.getMetrics(font, fontSize);
-      FontResolutionCache.add(
-        (family, Weight.Normal, false),
-        font,
-        fontCache,
-      );
+      FontResolutionCache.add((family, weight, false), font, fontCache);
 
       let maybeItalicFont =
         family
-        |> Revery_Font.Family.toSkia(~italic=true, Weight.Normal)
+        |> Revery_Font.Family.toSkia(~italic=true, weight)
         |> Revery.Font.FontCache.load;
 
       switch (maybeItalicFont) {
       | Ok(italicFont) =>
         if (isSameFamily(font, italicFont)) {
           FontResolutionCache.add(
-            (family, Weight.Normal, true),
+            (family, weight, true),
             italicFont,
             fontCache,
           );
         }
       | Error(msg) =>
+        // XXX: Update error message with weight ?
```

Comment:
Adding the weight here is a good idea! It would be nice to log it out in this message, like:
            `"Unable to load italic variant of %s with weight %s : %s",`

And we can use the `Revery.Font.Weight.show(weight)` to get a string to log there